suite/runsuite_wrapper.pl

```diff
@@ -341,6 +341,7 @@
                                    );
             # FIXME i#2417: fix flaky/regressed AArch64 tests
             %ignore_failures_64 = ('code_api|linux.sigsuspend' => 1,
+                                   'code_api|linux.thread-reset' => 1,
```

Comment:
Please point at a filed issue in a comment.  It seems best to not have this inside this huge PR?  Can we separate this out?

Reply:
Created https://github.com/DynamoRIO/dynamorio/issues/6741 and added comment.